data/partials/home.yaml

```diff
@@ -108,6 +108,10 @@ nav_sections:
         link: database_monitoring/
         icon: database-2
         desc: Explore enriched dashboards, query metrics, and query samples
+      - title: Data Streams Monitoring
+        link: data-streams/
```

Comment:
Check this link? Clicking on the tile in the preview gives me a 404.
Preview: https://docs-staging.datadoghq.com/esther/docs-5420-DSM-tile/